RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chrome.java

```diff
@@ -546,6 +548,20 @@ private void getDownload() {
                 if (bbart != null) {
                     bbartifacts.add(bbart);
                 }
+                
+                // find the downloaded file and create a TSK_DOWNLOAD_SOURCE for it..
+                try {
+                    for (AbstractFile downloadedFile : fileManager.findFiles(dataSource, FilenameUtils.getName(fullPath), FilenameUtils.getPath(fullPath))) {
+                        BlackboardArtifact downloadSourceArt =  downloadedFile.newArtifact(BlackboardArtifact.ARTIFACT_TYPE.TSK_DOWNLOAD_SOURCE);
+                        downloadSourceArt.addAttribute(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_URL,
+                            NbBundle.getMessage(this.getClass(), "Chrome.parentModuleName"),
```

Comment:
Change Chrome.parentModuleName to RecentActivityExtracterModuleFactory.getModuleName()